components/wso2is.key.manager.operations.endpoint/src/main/resources/keymanager-operations.yaml

```diff
@@ -305,6 +305,87 @@ paths:
               $ref: '#/definitions/Error'
         tags:
         - OAuth2 DCR
+
+  ######################################################
+  # The "Change Owner" of the oauth application
+  ######################################################
+  /dcr/{clientId}/change-owner:
+    post:
+      description: |
+        This operation is used to change the owner of an Application.
+      x-wso2-request: |
+        curl -k -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H
+                "Content-Type: application/json" -d '{
+                 "ext_application_owner": "admin"}'
+                "https://localhost:9443/keymanager-operations/dcr/change-owner?client_id=s6BhdRkqt3"
+      x-wso2-response: "HTTP/1.1 200 OK"
+      summary: Change Application Owner
+      parameters:
```

Comment:
Shall we add the operationId here?

Reply:
@Meruja which operation Id you mean?

Reply:
this operationId :D 
https://github.com/wso2/carbon-apimgt/blob/239c6d06bdd1a7725054013d9bd31ff77f3aaa69/components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml#L6563

Reply:
Adding with b29bcf3b4a8ec9b78b54d2cf3cb4c7b68fe9518f

Reply:
Done the changes with b29bcf3b4a8ec9b78b54d2cf3cb4c7b68fe9518f